src/S3/MultipartUploader.php

```diff
@@ -95,7 +95,7 @@ public static function getStateFromService(
      */
     public function __construct(S3Client $client, $source, array $config = [])
     {
-        parent::__construct($client, $source, $config + [
+        parent::__construct($client, $source, array_change_key_case($config) + [
```

Comment:
I don't use that function very often, but when you need it, it's just what the doctor ordered.